native/src/modules/app/components/Navigator.js

```diff
@@ -110,7 +111,7 @@ const Navigator = (props: PropsType) => {
         throw Error('The contentLanguage has not been set correctly by I18nProvider!')
       }
 
-      if (!buildConfig().featureFlags.introSlides) {
+      if (!buildConfig().featureFlags.introSlides && !introShown) {
         await appSettings.setIntroShown()
         await appSettings.setSettings({
           errorTracking: false,
```

Comment:
Should we set this to true? I.e. should we enable sentry error reports per default for Aschaffenburg? I am inclined to say yes as this is apparently not a data privacy issue.

Reply:
makes sense to me to set it to true.